src/components/GeoMap.js

```diff
@@ -423,9 +423,9 @@ export default class GeoMap {
 
       dataset.forEach(d => {
         d.rollupValue[attrKey] = attrValue.rollup(d.values);
-      });
+    });
 
-      const scaleDomain = d3.extent(dataset, d => d.rollupValue[attrKey]);
+      const scaleDomain = d3.extent(dataset, d => Number(d.rollupValue[attrKey]));
```

Comment:
Not sure why this would affect the sorting on the table.
We should maybe use `dataParse` to parse it as a number so that the sort for the table will work as well.